src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/custom.py

```diff
@@ -97,3 +98,90 @@ def _validate_start_time(start_time, end_time):
         raise ValueError("start_time '{}' is later than Now {}.".format(start_time, now))
 
     return result_time
+
+
+def list_activity_logs(client, correlation_id=None, resource_group=None, resource_uri=None,
```

Comment:
Ahaa then i've been calling it wrong at all the places. Let me update that to `resource_id`